lib/jose-jwe-decrypt.js

```diff
@@ -22,9 +22,10 @@
  * @param key_promise    Promise<CryptoKey>, either RSA or shared key
  */
 JoseJWE.Decrypter = function(cryptographer, key_promise) {
-  this.cryptographer = cryptographer;
-  this.key_promise = key_promise;
-  this.headers = {};
+  var that = this;
```

Comment:
minifiers can reduce it to a single letter thus achieving a better compression rate on the minified file